polynote-kernel/src/main/scala/polynote/kernel/interpreter/State.scala

```diff
@@ -127,8 +127,8 @@ trait State {
     if (predicate(this)) {
       return this
     }
-    var s = this
-    while (s != Root && !predicate(s.prev)) {
+    var s = this.prev
```

Comment:
So it just skips the tip state? What if the predicate holds for `this`?

Reply:
That's covered by ` if (predicate(this)) {` at 127 :)